### PR TITLE
[TASK] assure typo3temp/Cache is writeable command

### PR DESCRIPTION
**Pull request by achimfritz**
Surf Task/Application  kommt gleich

cc: @lars85

### CODE REVIEW

File: Classes/Command/CacheApiCommandController.php [hunk 1]
```diff
@@ -89,6 +89,23 @@ public function clearAllCachesCommand($hard = false) {
...
+	 * @throws Etobi\Coreapi\Service\Exception
+	 */
+	public function assureCacheDirectoryIsWriteableCommand() {
+		try {
+			$this->cacheApiService->assureCacheDirectoryIsWriteable();
+			$message = 'Ok, typo3temp/Cache is writeable';
+			$this->logger->info($message);
+			$this->outputLine($message);
+		} catch (\UnexpectedValueException $e) {
+			$message = 'Ok, typo3temp/Cache not exists';
```
Comment (ervaude) [inline Classes/Command/CacheApiCommandController.php]: Lets use real english for the message:
"typo3temp/Cache does not exist."

=)

File: Classes/Service/CacheApiService.php [hunk 2]
```diff
@@ -99,13 +99,34 @@ public function initializeObject() {
 	 * Clear all caches.
 	 *
 	 * @param bool $hard
+	 * @throws Etobi\Coreapi\Service\Exception
```
Comment (ervaude) [inline Classes/Service/CacheApiService.php:102]: \ Etobi\Coreapi\Service\Exception
or 
Exception

File: Classes/Command/CacheApiCommandController.php (hunk 1 above)
Comment (achimfritz) [inline Classes/Command/CacheApiCommandController.php]: Spiesser

File: Classes/Service/CacheApiService.php (hunk 2 above)
Comment (achimfritz) [inline Classes/Service/CacheApiService.php:102]: fixed

**Comment by ervaude** [comment]
:+1:

**Pull request merged**